.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+<!-- Replace <`issueNumber`> with the Issue number to link it with this PR -->
+<!-- Example: #1 links this PR to the first Issue-->
+resolves #`issueNumber`
+
+## Checklist
+
+- [ ] You've followed the code structure 
+- [ ] You've linked the PR to correct Issue
+
+## Proposed Changes
+
+  -
+  -
+  -
```

Comment:
Add a commented block which says that one can add checkboxes for proposed changes and update the message as they have committed the changes.